doc/cookbook/source/examples/classifier/LDA.rst

```diff
@@ -0,0 +1,54 @@
+=============================
+Linear Discriminant Analysis
+=============================
+
+LDA learns a linear classifier and requires examples to be CDenseFeatures. The learned linear classification rule is optimal under the assumption that both classes a gaussian distributed with equal co-variance. To find a linear separation :math:`{\bf w}` in training, the in-between class variance is maximized and the within class variance is minimized.
```

Comment:
LDA attempts to find a projection matrix that maximally discriminates the provided classes.